typegate/tests/e2e/typegraph/typegraphs/deno/graphql.ts

```diff
@@ -0,0 +1,19 @@
+import { g, t, typegraph } from "../../../../../../typegraph/deno/src/mod.ts";
+import { GraphQLRuntime } from "../../../../../../typegraph/deno/src/runtimes/graphql.ts";
+import * as effects from "../../../../../../typegraph/deno/src/effects.ts";
+
```

Comment:
you can put your test in the runtime/deno directly, I would keep e2e for more complex scenarios (like multiple pushes)